### PR TITLE
Upgrade composer dependencies to be php8 compatible

### PR DESCRIPTION
**Pull request by rvdbogerd**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,11 +25,13 @@
     "jms/serializer": "^1.6|^2.0|^3.0",
 
     "goetas-webservices/xsd2php-runtime": "^0.2.11",
-    "goetas-webservices/soap-common": "^0.2.0"
+    "goetas-webservices/soap-common": "dev-master",
```
Comment (goetas) [inline composer.json:28]: i have released a new version for this https://github.com/goetas-webservices/soap-common/releases/tag/0.2.2

File: composer.json [hunk 2]
```diff
@@ -25,11 +25,13 @@
...
+    "goetas-webservices/soap-reader": "^0.3.3",
+    "goetas-webservices/xsd2php": "dev-master"
```
Comment (goetas) [inline composer.json:30]: i have released a new version for this https://github.com/goetas-webservices/xsd2php/releases/tag/0.4.7

**Review by goetas — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (rvdbogerd) [reply composer.json]: Thank you so much!

File: composer.json (hunk 2 above)
Comment (rvdbogerd) [reply composer.json]: and again, ty :)

**Pull request merged**